wrappers/services.go

```diff
@@ -953,6 +953,10 @@ func genServiceNames(snap *snap.Info, appNames []string) []string {
 	return names
 }
 
+func GenServiceNames(snap *snap.Info, appNames []string) []string {
```

Comment:
Just make the original function exported (there will be just 2 or 3 places affected by this).

Reply:
agreed with this suggestion, also feel free to include my patch in this PR or to push to my PR so we can have the spread test which shows it works alongside the fix

Reply:
Done! I'm off for holidays, so feel free to adjust this as needed :-)